x/consensus/proto/buf.gen.pulsar.yaml

```diff
@@ -9,6 +9,7 @@ managed:
       - buf.build/cosmos/cosmos-proto
     override:
       buf.build/cosmos/cosmos-sdk: cosmossdk.io/api
+      buf.build/tendermint/tendermint: buf.build/gen/go/tendermint/tendermint/protocolbuffers/go
```

Comment:
I'm asking a dumb question, but why these changes only in consensus and staking modules ?

Reply:
Not a dumb question, this is because only consensus and staking use comet proto types.
Before we generated them in api, but now comet has its protos on buf, this isn't needed.

Reply:
thanks for the clarification julien